icicle-compiler/src/Icicle/Sea/Preamble.hs

```diff
@@ -26,16 +26,37 @@ import           X.Data.FileEmbed (embedWhen)
 seaPreamble :: Doc
 seaPreamble
  = vsep
- $ fmap (uncurry seaOfExternal) files
+ ( fmap (uncurry seaOfExternal) externals <> fmap (uncurry seaOfHeader) files )
  where
   files
    = List.sortBy (compare `on` fst)
    $ $(embedWhen ((== ".h") . takeExtension) "data/sea/")
+  externals
+   = let base    = $(embedWhen ((== "anemone_base.h"))    "data/csrc/anemone")
+         mempool = $(embedWhen ((== "anemone_mempool.h")) "data/csrc/anemone")
```

Comment:
I wanted to be explicit in case those headers don't have same structure as icicle sea headers (ie the file doesn't start with the #includes)